src/custom/state/affiliate/reducer.ts

```diff
@@ -1,17 +1,19 @@
 import { createReducer } from '@reduxjs/toolkit'
-import { updateAppDataHash, updateReferralAddress } from './actions'
-import { APP_DATA_HASH } from 'constants/index'
+import { dismissNotification, updateAppDataHash, updateReferralAddress } from './actions'
+import { APP_DATA_HASH, IS_NOTIFICATION_CLOSED } from 'constants/index'
 
 export interface AffiliateState {
   referralAddress?: {
     value: string
     isValid: boolean
   }
   appDataHash?: string
+  isNotificationClosed?: boolean
 }
 
 export const initialState: AffiliateState = {
   appDataHash: APP_DATA_HASH,
+  isNotificationClosed: IS_NOTIFICATION_CLOSED,
```

Comment:
Yes, that's the direction we want to go!

We just need 2 more dimensions: `account` and `reason/type of notification`

With the current setup, if dismissed once, the pop-up will never be displayed again for other accounts or reasons